README.md

```diff
@@ -0,0 +1,29 @@
+# org.unmojang.FjordLauncher
```

Comment:
no user will every see this.

Reply:
Sure, I can remove the README. FWIW, if I Google "prismlauncher flatpak", the third result is https://github.com/flathub/org.prismlauncher.PrismLauncher. Obviously that won't be the case if the README is empty, but if the README is there, it could be a good place to put Flatpak-specific installation notes.

Reply:
I was just clarifying the expectation here.